torchgen/dest/register_dispatch_key.py

```diff
@@ -56,6 +56,8 @@ def gen_registration_headers(
             headers.append("#include <ATen/hip/EmptyTensor.h>")
         else:
             headers.append("#include <ATen/cuda/EmptyTensor.h>")
+    elif backend_index.dispatch_key == DispatchKey.MPS:
+        headers.append("#include <ATen/native/mps/TensorFactory.h>")
```

Comment:
Minor call-out: it might be worth trying to make the file structure more similar to what we have for cpu/cuda? For cuda, we have `aten/src/ATen/cuda/EmptyTensor.h` defining an `at::detail::empty{_strided}_cuda`. Maybe the MPS equivalent should go in `aten/src/ATen/mps/EmptyTensor.h`?

Reply:
Done.